e2e/features/bulk-scanning/create-appeal.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Create an bulk-scanned appeal
+
+  Scenario: Should end up in "appeal created" state when all mandatory fields are present
+    Given I am signed in as a Case Officer
+    And I have a bulk-scanned document with all mandatory fields
+    When I choose the next step "Create new case from exception"
+    Then the case should be in "Case created" state
```

Comment:
Is Case created the right name for the state? Is it not 'Appeal Created'?

Reply:
thats an interesting question, I might have an old version of the definition

Reply:
Case Created is the Bulk Scan definition name. We need to also be testing for 'Appeal Created' in SSCS CCD. Will find out today how we can link the 2 cases so you can easily find it for this test